GitCommands/StringExtensions.cs

```diff
@@ -232,5 +232,15 @@ public static string ShortenTo([CanBeNull] this string str, int maxLength)
 
             return str.Substring(0, maxLength - 3) + "...";
         }
+
+        /// <summary>
+        /// Uses the method IndexOf to check if the <paramref name="other"/> is contained inside the <paramref name="str"/>
+        /// </summary>
+        /// <returns>true if the <paramref name="other"/> is inside of <paramref name="str"/></returns>
+        public static bool Contains([NotNull]this string str, [NotNull] string other,
+            StringComparison stringComparison = StringComparison.InvariantCultureIgnoreCase)
```

Comment:
If you want to use a default, it should be `StringComparison.Ordinal` which matches the `String.Contains(String)` instance method behaviour.

Reply:
Having the default be case-insensitive would be surprising relative to the rest of the framework.